io.github.cnruby.main_139.json

```diff
@@ -0,0 +1,35 @@
+{
+  "app-id": "io.github.cnruby.main_139",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "19.08",
+  "sdk": "org.freedesktop.Sdk",
+  "command": "main_139",
+  "modules": [
+      {
+          "name": "main_139",
+          "only-arches": [
+            "x86_64"
+          ],
+          "cmake": true,
+          "buildsystem": "cmake-ninja",
```

Comment:
do you want cmake or cmake-ninja? in either case the cmake: true is unecessary.

Reply:
OK, thanks